be/src/vec/functions/function_string.h

```diff
@@ -1437,4 +1438,22 @@ class FunctionReplace : public IFunction {
     }
 };
 
+struct ReverseImpl {
+    static Status vector(const ColumnString::Chars& data, const ColumnString::Offsets& offsets,
+                         ColumnString::Chars& res_data, ColumnString::Offsets& res_offsets) {
+        auto rows_count = offsets.size();
```

Comment:
impl codes put into cpp file

Reply:
As we discussed, keep the code style firstly.